framework/src/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -273,13 +273,17 @@ private void annotateArrayAsArray(AnnotatedArrayType result, NewArrayTree node,
     }
 
     /**
-     * The type of a NewClassTree is the type of the Identifier
-     * plus any explicit annotations (including polymorphic qualifiers)
-     * on the constructor.
+     * Creates an AnnotatedDeclaredType for the NewClassTree and adds one of:
+     * <ul>
+     *   <li>explicit annotations from the new class expression ({@code new @HERE MyClass()} ), or</li>
+     *   <li>explicit annotations from the declaration of the class ({@code @HERE class MyClass {}} ), or</li>
```

Comment:
This case wasn't in the old documentation and isn't obvious from the implementation. Where in the body are these added?